okhttp-tests/src/test/java/com/squareup/okhttp/internal/http/RouteSelectorTest.java

```diff
@@ -379,7 +316,8 @@
     client.setProxy(Proxy.NO_PROXY);
     RouteSelector routeSelector = RouteSelector.get(address, httpsRequest, client);
 
-    dns.inetAddresses = makeFakeAddresses(255, 1);
+    final int numberOfAddresses = 2;
```

Comment:
Typically I do this for method-scoped constants. i.e. it is a constant that I've pulled out to give it a clear name and/or because it's used in multiple places that must agree, but which don't make sense outside of the test. Where coding conventions permit I make them upper-case too. By marking it final the reader can mentally dismiss the chance of it changing.

In this case, the 1 in the code that was replaced was unclear as to its meaning.

This seemed preferable to having:

2 /* numberOfAddresses */ in a couple of places.